examples/io/read_older_nexrad_data_aws.py

```diff
@@ -0,0 +1,77 @@
+"""
+==================================================================
+Reading Older NEXRAD Data and Fixing Latitude and Longitude Issues
+==================================================================
+"""
+print(__doc__)
+
+
+# Author: Zachary Sherman (zsherman@anl.gov)
+# License: BSD 3 clause
+
+######################################
+# Import our required packages.
+
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+
+import pyart
+
+######################################%%
+# Read older NEXRAD Level 2 Data
+# ------------------------------
+#
+# Older NEXRAD files have the tendency to not contain some of the required
+# metadata for Py-ART's NEXRAD reader. This usually results in missing
+# latitude and longitude data, so after reading with Py-ART, both coordinates
+# have a value of 0. This example, we will show how to properly read in an
+# older NEXRAD file.
+#
+# First we want to get an older file from amazon web service:
+#
+#    ``s3://noaa-nexrad-level2/year/month/date/radarsite/{radarsite}{year}{month}{date}_{hour}{minute}{second}.gz``
+# 
+# Where in our case, we are using a sample data file from Handford, CA (KHNX)
+# on July 24, 2006, at 0203:38 UTC. This means our path would look like this:
+
+aws_nexrad_level2_file = "s3://noaa-nexrad-level2/2006/07/24/KHNX/KHNX20060724_020338.gz"
+
+# Note: Older files do not contain the 'V06' but instead '.gz' in the AWS path.
```

Comment:
Can you specify older than 2008? Which I believe is the year they switched the format.